sources/ua/ukraine-regions.hjson

```diff
@@ -0,0 +1,137 @@
+{
+    versions: '>=1'
+    type: http
+    production: true
+    wikidata: Q3348196
+    note: Oblast of Ukraine
+    data: https://sophox.org/regions/geojson.json
+    attribution: [
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+            type: id
+            name: iso_3166_2
+            desc: ISO-3166-2 identifier
+        }
+        {
+            type: property
+            name: label_en
+            desc: Subdivision name (English)
+        }
+        {
+            type: property
+            name: label_uk
+            desc: Subdivision name (Ukranian)
+        }
+        {
+            type: property
+            name: label_ru
+            desc: Subdivision name (Russian)
+        }
+    ]
+    query: {
+        sparql:
+            '''
+            #defaultView:MapRegions
+            SELECT (if(bound(?id2),?id2,?id1) as ?id)
+                   ?iso_3166_2
+                   ?label_en ?label_uk ?label_ru
+            WHERE {
+              SERVICE <https://query.wikidata.org/sparql> {
+
+                SELECT ?id1
+                       (SAMPLE(?id2) as ?id2)
+                       (SAMPLE(?iso_3166_2) as ?iso_3166_2)
+                       (SAMPLE(?label_en) as ?label_en)
+                       (SAMPLE(?label_uk) as ?label_uk)
+                       (SAMPLE(?label_ru) as ?label_ru)
+                WHERE {
+                    # List of regions, whose sub-regions we want.
+                    VALUES ?entity { wd:Q212 }
+
+                    # P150 = "contains administrative territorial entity"
+                    ?entity wdt:P150 ?id1 .
+                    ?id1 wdt:P300 ?iso_3166_2
+
+                    BIND(if(?id1=wd:Q7525, wd:Q4456205, ?id1)  AS ?id2)
+
+                    # Labels
+                    OPTIONAL { ?id1 rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
+                    OPTIONAL { ?id1 rdfs:label ?label_uk . FILTER(LANG(?label_uk) = "uk") }
+                    OPTIONAL { ?id1 rdfs:label ?label_ru . FILTER(LANG(?label_ru) = "ru") }
+                }
+                # remove possible ID duplicates
+                GROUP BY ?id1
+              }
+            }
+            ORDER BY ?iso_3166_2
+            '''
+    }
+    name: ukraine_regions
+    legacyIds: [
+      Ukraine Regions
+    ]
+    humanReadableName: {
+      ar: أوبلاستات أوكرانيا
+      ast: Oblast d'Ucraína
+      be: вобласць Украіны
+      be-tarask: вобласьць Украіны
+      bs: Oblasti u Ukrajini
+      ca: Oblast d'Ucraïna
+      cs: Oblast
```

Comment:
This label is too generic and does not mention the country. Unless you know the correct translation, I suggest removing it. 
```suggestion

```